nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/test/resources/server.properties

```diff
@@ -21,25 +22,27 @@ broker.id=0
 
 ############################# Socket Server Settings #############################
 
-# The port the socket server listens on
+# The address the socket server listens on. It will get the value returned from 
```

Comment:
This is a bit confusing.
Is `listeners` a new property while `port` is still a valid alternative? Probably would be better to clarify in the comments.

Reply:
The `server.properties` file is an original Kafka 2.0 server config file with minimal changes. I would not edit the comments in it.
I also removed the port=9092 property which was not necessary (and deprecated).